lib/data_hygiene/publishing_api_sync_check.rb

```diff
@@ -41,18 +44,19 @@ def to_row
     class Failure
       attr_reader :base_path, :failed_expectations
 
-      def initialize(base_path:, failed_expectations:, content_store:)
+      def initialize(whitehall_model_id:, base_path:, failed_expectations:, content_store:)
```

Comment:
Better to call this `record_id`. A "model" is the class, a "record" is the instance.